torchdynamo/codegen.py

```diff
@@ -163,6 +174,13 @@ def __call__(self, value, allow_cache=True):
 
         self.top_of_stack = value
 
+    def _add_to_spec(self, value, add_open):
+        if isinstance(value, (ListVariable, TupleVariable, ConstDictVariable)):
+            if add_open:
```

Comment:
Should these just be two functions? (open and close)

Reply:
I had it that way, but the isinstance check felt grosser to duplicate, no strong opinions